src/accountExporter.js

```diff
@@ -50,7 +51,7 @@ var _addProviderUserInfo = function(providerInfo, arr, startPos) {
 };
 
 var _transUserToArray = function(user) {
-  var arr = Array(26).fill("");
+  var arr = Array(27).fill("");
```

Comment:
This magic number pile kinda makes me want to give up computers ... but not worth changing now.